content/en/docs/concepts/services-networking/service.md

```diff
@@ -91,32 +91,32 @@ spec:
     targetPort: 9376
 ```
 
-This specification will create a new Service object named “my-service” which
-targets TCP port 9376 on any Pod with the `"app=MyApp"` label.
+This specification creates a new Service object named “my-service”, which
+targets TCP port 9376 on any Pod with the `app=MyApp` label.
 
-This Service will also be assigned an IP address (sometimes called the "cluster IP"),
-which is used by the service proxies
+This Service also assigns an IP address (sometimes called the "cluster IP"),
+which is used by the Service proxies
```

Comment:
nit: This specification creates a new Service named "my-service", ...

nit, could the link be cleaned up a bit:
This Service also assigns an IP address, or "cluster IP", which is used by the [Virtual IPs and service proxies](#virtual-ips-and-service-proxies).